test/metabase/query_processor_test/timezones_test.clj

```diff
@@ -248,7 +279,7 @@
       (for [i (range 366)]
         [(u.date/add start-date :day i)])]]))
 
-(defmethod driver/database-supports? [:driver ::extract-week-of-year-us]
+(defmethod driver/database-supports? [::driver ::extract-week-of-year-us]
```

Comment:
Is this change accidental? `:driver` is the parent of all drivers basically the same as saying `:default` here if vanilla multimethods let you do that

Reply:
Not accidental. This change fixes the fact that we weren't testing `:week-of-year` in the following test:
https://github.com/metabase/metabase/blob/23abd17adbd7b7ca869d81f727a5d36f5d0082d2/test/metabase/query_processor_test/timezones_test.clj#L310-L312

And no I don't think vanilla multimethods lets you use `:default` like that. 

```
(defmulti x (fn [a b] [a b]))
(defmethod x [:a :b] [_ _] true) ; this doesn't
(defmethod x :default [_ _] true) ; this works
(defmethod x [:default :b] [_ _] true) ; this doesn't work
```

Reply:
I've split this change off into another PR: https://github.com/metabase/metabase/pull/44583